examples/common_vmseries/main_test.go

```diff
@@ -0,0 +1,46 @@
+package common_vmseries
+
+import (
+	"os"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+
+	"github.com/PaloAltoNetworks/terraform-azure-vmseries-modules/go/testskeleton"
```

Comment:
I propose to import external package (instead of using internal folder `go`):

```suggestion
	"github.com/PaloAltoNetworks/terraform-modules-vmseries-tests-skeleton/pkg/testskeleton"
```

Reply:
fixed